cmd/activator/request_log.go

```diff
@@ -29,11 +30,20 @@ import (
 
 func updateRequestLogFromConfigMap(logger *zap.SugaredLogger, h *pkghttp.RequestLogHandler) func(configMap *corev1.ConfigMap) {
 	return func(configMap *corev1.ConfigMap) {
-		newTemplate := configMap.Data["logging.request-log-template"]
-		if err := h.SetTemplate(newTemplate); err != nil {
-			logger.Errorw("Failed to update the request log template.", zap.Error(err), "template", newTemplate)
+		var newTemplate string // default is an empty template, disables logging
+		obsconfig, err := metrics.NewObservabilityConfigFromConfigMap(configMap)
+		if err != nil {
+			logger.Errorw("Failed to get observability configmap.", zap.Error(err), "configmap", configMap)
```

Comment:
probably better to do this as a guard clause at the top with a return, then the rest of the method can be out-dented

Reply:
refactored